analytics-integrations/apptimize/src/main/java/com/segment/analytics/internal/ApptimizeIntegration.java

```diff
@@ -0,0 +1,94 @@
+package com.segment.analytics.internal.integrations;
+
+import android.Manifest;
+import android.content.Context;
+import com.apptimize.Apptimize;
+import com.segment.analytics.Traits;
+import com.segment.analytics.ValueMap;
+import com.segment.analytics.internal.AbstractIntegration;
+import com.segment.analytics.internal.model.payloads.IdentifyPayload;
+import com.segment.analytics.internal.model.payloads.ScreenPayload;
+import com.segment.analytics.internal.model.payloads.TrackPayload;
+
+import static com.segment.analytics.Analytics.LogLevel;
+import static com.segment.analytics.internal.Utils.hasPermission;
+import static com.segment.analytics.internal.Utils.isNullOrEmpty;
+
+/**
+ * Apptimize allows you to instantly update your native app without waiting for
+ * App or Play Store approvals, and easily see if the change improved the app
+ * with robust A/B testing analytics.
+ *
+ * @see <a href="http://www.apptimize.com/">Apptimize</a>
+ * @see <a href="https://segment.com/docs/integrations/apptimize/">Apptimize Integration</a>
+ */
+public class ApptimizeIntegration extends AbstractIntegration<Void> {
+  static final String APPTIMIZE_KEY = "Apptimize";
+
+  @Override public void initialize(Context context, ValueMap settings, LogLevel logLevel)
+      throws IllegalStateException {
+    if (!hasPermission(context, Manifest.permission.INTERNET)) {
```

Comment:
Our SDK can't be initialized without the internet permission either, so you don't need to check for this.